lib/smart_answer_flows/coronavirus-business-reopening/outcomes/_social_distancing.erb

```diff
@@ -75,7 +75,7 @@ You should:
 
 
 <% if calculator.sector?("homes") %>
-  ### Working in other people’s homes
+  ## Working in other people’s homes
```

Comment:
Again - I think these should be h3. They are headers for sub-clauses of the main section headed "How to ensure social distancing on site"